source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -119,6 +119,11 @@ class StreetNetworkSerializer(OutsideServiceCommon):
     def get_timeout(self, obj):
         return obj.get('timeout', None)
 
+    backend_class = MethodField(schema_type=str, display_none=False)
```

Comment:
You should put it above `get_timenout()` to improve readability

Reply:
`get_backend_class()` to or just the field?

Reply:
just the field, to group them